rpi.py

```diff
@@ -20,8 +24,17 @@ def parse_output(pattern, default, args):
 
 
 def cpu_temp():
-    return parse_output(r'temp=(\S*)\'C', '0', ['vcgencmd', 'measure_temp'])
+    t_str = parse_output(r'temp=(\S*)\'C', '0', ['vcgencmd', 'measure_temp'])
```

Comment:
Defaulting to '0' does not look right to me. I believe that None should be default here.

Reply:
Fixed.  `ip_address` calculated in the same way (default=None instead of '?')

Reply:
Dunno. It could probably be an empty string or just be like it is.